boards/shields/x_nucleo_wb05kn1/x_nucleo_wb05kn1_spi.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>;	      /* D10 */
+
+	bluenrg-lp@0 {
```

Comment:
I think this needs to be updated following https://github.com/zephyrproject-rtos/zephyr/pull/72323.

Reply:
Done.